VMEncryption/main/Common.py

```diff
@@ -57,6 +57,8 @@ class CommonVariables:
     default_block_size = 52428800
     min_filesystem_size_support = 52428800 * 3
     default_file_system = 'ext4'
+    format_supported_file_systems = ['ext4', 'ext3', 'ext2', 'xfs', 'btrfs']
```

Comment:
will there be a corresponding update to public documentation that needs to be made along with this change

Reply:
I don't think so. This is just a list of formats we are comfortable running mkfs.[format] with.